config/logstash.yml

```diff
@@ -181,25 +181,6 @@
 #
 # api.auth.basic.password_policy.mode: WARN
 #
-# ------------ Module Settings ---------------
-# Define modules here.  Modules definitions must be defined as an array.
-# The simple way to see this is to prepend each `name` with a `-`, and keep
-# all associated variables under the `name` they are associated with, and
-# above the next, like this:
-#
-# modules:
-#   - name: MODULE_NAME
-#     var.PLUGINTYPE1.PLUGINNAME1.KEY1: VALUE
-#     var.PLUGINTYPE1.PLUGINNAME1.KEY2: VALUE
-#     var.PLUGINTYPE2.PLUGINNAME1.KEY1: VALUE
-#     var.PLUGINTYPE3.PLUGINNAME3.KEY1: VALUE
-#
-# Module variable names must be in the format of
-#
-# var.PLUGIN_TYPE.PLUGIN_NAME.KEY
-#
-# modules:
-#
```

Comment:
The `Cloud Settings` below still refer to module specific settings `var.elasticsearch.hosts` and `var.kibana.host`, etc. 
If these settings are not relevant at all, we should remove them. If they are relevant to central management and internal collection, we should move and rework them

Reply:
deleted